### PR TITLE
Export keypad-context separately to avoid bundle bloat

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
<Add your summary here>

Issue: LEMS-2246

## Test plan:

### CODE REVIEW

**Comment by jeresig** [comment]
Great! So a question I have is that what's being exported here will be a different file than the one that's referenced internally inside Math Input. Is that a problem? It's possible that the actual context object may be different between this exported one and the "internal" one?

File: packages/keypad-context/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "source": "src/index.ts",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
+    },
+    "dependencies": {},
+    "devDependencies": {},
+    "peerDependencies": {},
```
Comment (jeremywiebe) [inline packages/keypad-context/package.json:29]: We should be listing 'react' here as a peer dep.

File: packages/keypad-context/src/keypad-context.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export function StatefulKeypadContextProvider(props: Props) {
     // whether or not to display the keypad
     const [keypadActive, setKeypadActive] = useState<boolean>(false);
     // used to communicate between the keypad and the Renderer
-    const [keypadElement, setKeypadElement] = useState<KeypadAPI | null>();
+    const [keypadElement, setKeypadElement] = useState<any>();
```
Comment (jeremywiebe) [inline packages/keypad-context/src/keypad-context.tsx:35]: 😬  `any` ?

File: packages/keypad-context/types [hunk 3]
```diff
@@ -0,0 +1 @@
+../../types/
```
Comment (jeremywiebe) [inline packages/keypad-context/types:1]: Github doesn't show this well, but wanted to confirm this is a symlink.

File: packages/math-input/src/components/__tests__/integration.test.tsx [hunk 4]
```diff
@@ -1,3 +1,9 @@
+import {
+    KeypadContext,
+    StatefulKeypadContextProvider,
+    // @ts-expect-error it can't find type declaration
+    // eslint-disable-next-line monorepo/no-internal-import
```
Comment (jeremywiebe) [inline packages/math-input/src/components/__tests__/integration.test.tsx]: Is this error still happening?

**Review by jeremywiebe — APPROVED** [review]
Let's hope this works! :)

File: packages/keypad-context/src/keypad-context.tsx (hunk 2 above)
Comment (handeyeco) [reply packages/keypad-context/src/keypad-context.tsx:35]: Yeah. I tried to move types from `@khanacademy/math-input` and things quickly spiraled. I think short-term it's good to have this separate package, but long-term we need to find a better solution.

**Pull request merged**